Rabbitmq Weak Credentials Plugin

leonardo-doyensec opened the pull request:
Created a plugin for testing weak credentials in rabbitmq management portal

dawidg-doyen commented on google/detectors/credentials/generic_weak_credential_detector/src/test/java/com/google/tsunami/plugins/detectors/credentials/genericweakcredentialdetector/testers/rabbitmq/RabbitMQCredentialTesterTest.java:
```diff
@@ -0,0 +1,160 @@
+
+  private static final String WEAK_CRED_AUTH_1 = "Basic dXNlcjoxMjM0";
+  private static final String WEAK_CRED_AUTH_2 = "Basic cm9vdDpwYXNz";
+  private static final ServiceContext.Builder rabbitmqServiceContext =
```
use capitals if it's static

dawidg-doyen commented on google/detectors/credentials/generic_weak_credential_detector/src/main/java/com/google/tsunami/plugins/detectors/credentials/genericweakcredentialdetector/testers/rabbitmq/RabbitMQCredentialTester.java:
```diff
@@ -0,0 +1,204 @@
+  }
+
+  // Checks if the response body contains elements of a rabbitmq management page - custom
+  // fingerprinting phase
```
Inconsistent use of `fingerprinting`, use `fingerprint` form as you used down below.

dawidg-doyen commented on google/detectors/credentials/generic_weak_credential_detector/src/test/resources/com/google/tsunami/plugins/detectors/credentials/genericweakcredentialdetector/testers/rabbitmq/testdata/successfulAuthdResponse.json:
```diff
@@ -0,0 +1 @@
+{"name":"","tags":["administrator"]}
```
should the name be empty or should it contain name i.e. 'admin' ?

dawidg-doyen reviewed (changes requested):
Done with the review. Rest of the comments provided via chat. Remember to run the checkstyle linter at the end.

leonardo-doyensec commented:
Updated everything as you requested. Regarding the empty username in the json file i've replaced it with a placeholder that at run-time will be updated with the current username
